server/migrations/20210309015328_add_labels.js

```diff
@@ -0,0 +1,18 @@
+
+exports.up = function(knex) {
+  return knex.schema.createTable('labels', (table) => {
+    table.increments('id').primary();
+    table.string('name');
+    table.timestamp('created_at').defaultTo(knex.fn.now());
+    table.timestamp('updated_at').defaultTo(knex.fn.now());
+  }).createTable('tasks_labels', (table) => {
+    table.integer('task_id');
+    table.integer('label_id');
+    table.timestamp('created_at').defaultTo(knex.fn.now());
+    table.timestamp('updated_at').defaultTo(knex.fn.now());
+  });
+};
+
```

Comment:
i'd recommend to always split such migrations to two files. so we would be more flexible then we need rollback

Reply:
done